src/store/services/account.ts

```diff
@@ -248,6 +258,7 @@ export const accountServiceSlice = createApi({
 export const {
     useSendOTPQuery,
     useGetUsersQuery,
+    useGetCampusSummeryByCampusIdQuery,
```

Comment:
Typographical error `useGetCampusSummeryByCampusIdQuery`, supposed to be `useGetCampusSummaryByCampusIdQuery`